### PR TITLE
BUGFIX: show validation errors for invalid electronic address

### PR DESCRIPTION
**Pull request by Pingu501**
Fixes: [#1807](https://github.com/neos/neos-development-collection/issues/1807)
Related: [#3841](https://github.com/neos/neos-development-collection/issues/3841)

### CODE REVIEW

**Review by bwaidelich — APPROVED** [review]
+1 by reading

File: Classes/Domain/Validator/ElectronicAddressValidator.php [hunk 1]
```diff
@@ -65,8 +66,10 @@ public function isValid($value)
             } else {
                 $result = $addressValidator->validate($value->getIdentifier());
                 if ($result->hasErrors()) {
+                    $identifierResult = $this->getResult()->forProperty('identifier');
                     foreach ($result->getErrors() as $error) {
-                        $this->addError($error->getMessage(), $error->getCode());
+                        /** @var Error $error */
```
Comment (kdambekalns) [inline Classes/Domain/Validator/ElectronicAddressValidator.php]: I'd move this before the `foreach`, so it's closer to where the variable first appears.

**Review by kdambekalns — APPROVED** [review]
Looks good by reading, apart from a nitpick.

File: Classes/Domain/Validator/ElectronicAddressValidator.php (hunk 1 above)
Comment (Pingu501) [reply Classes/Domain/Validator/ElectronicAddressValidator.php]: Yes looks better. Is fixed ;)

**Comment by bwaidelich** [comment]
Tested it now again and everything works as expected.

@Pingu501 Thanks for your first contribution :)

*Note:* Please don't force-push to PRs so we don't lose the history

**Comment by kdambekalns** [comment]
Released with https://github.com/neos/party/releases/tag/7.0.2 🎉

**Pull request merged**